[Docs] Add missing upgrade notes for Redis cache (#16197)
pimcore/pimcore: doc/23_Installation_and_Upgrade/07_Updating_Pimcore/12_V10_to_V11.md

### WYSIWYG-Editor
Please add all html tags with attributes you want to allow to the html_sanitizer like described [here](../../03_Documents/01_Editables/40_WYSIWYG.md)

### Redis Cache Configuration
If you are using `Redis` adapter for `Pimcore\Cache` then adapt the configuration as below:
```yaml
framework:
    cache:
        pools:
            pimcore.cache.pool:

>                 #tags: true remove it

Reviewer: "remove it" why?

Reviewer 2: #15625

Reviewer: So it should say "remove this line when upgrading" or some such thing?

Author: what do you mean? it is a section in upgrade notes, basically the steps to perform before upgrading to v11.

Reviewer: I don't know what I mean, that is probably because I don't really know what to do when reading these instructions. I'm basically asking for a clarification.

Author: I think then you got the answer in my last comment :wink:

Author: please follow the [discussion](https://github.com/pimcore/pimcore/issues/15605) here for the reason to remove it.

Reviewer: If the discussion is key to understanding what to do here maybe it should be linked in the documentation? After all, documentation is about being explicit about what to do and not rely on subtle knowledge gleaned by combing through Github.

>                 #tags: true remove it

Reviewer 3: ```suggestion
                #tags: true // this line needs to be removed
```

                public: true 
                default_lifetime: 31536000  # 1 year

>                 adapter: cache.adapter.redis_tag_aware #do not use pimcore.cache.adapter.redis_tag_aware

Reviewer 3: ```suggestion
                adapter: cache.adapter.redis_tag_aware # use this instead of pimcore.cache.adapter.redis_tag_aware
```
